### PR TITLE
fix: ephemeral builds

### PR DESCRIPTION
**Pull request by eschutho**
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by mistercrunch** [comment]
closing in favor of https://github.com/apache/superset/pull/31385

File: Dockerfile [hunk 1]
```diff
@@ -112,6 +112,7 @@ RUN --mount=type=bind,source=./docker,target=/docker \
       superset-frontend \
       apache_superset.egg-info \
       requirements \
+      {SUPERSET_HOME} \
```
Comment (eschutho) [inline Dockerfile:115]:
```suggestion
      ${SUPERSET_HOME} \
```

**Pull request closed without merging**